modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -236,6 +227,29 @@ export function FieldBase({
 		type === 'paragraph' ||
 		type === 'radio';
 	const showPopover = fieldName === 'inputMaskFormat';
+	const showFor =
+		type === 'text' ||
+		type === 'numeric' ||
+		type === 'image' ||
+		type === 'search_location';
+
+	const fieldDetails = getFieldDetails({
+		errorMessage,
+		hasError,
+		label,
+		required,
+		text,
+		tip,
+		warningMessage,
+	});
+
+	const fieldDetailsId = `${id ?? name}_fieldDetails`;
+
```

Comment:
Move the variable declarations of `fieldDetails` and `fieldDetailsId` bellow `harError` variable, on line 199.

Reply:
I moved there in order to have showFor after all show... variables, otherwise I need to move showFor up as well.

Regards.